### PR TITLE
csv-merger: Add OCS Toleration to metrics-exporter

### PR DESCRIPTION
**Pull request by jarrpa**
Signed-off-by: Jose A. Rivera <jarrpa@redhat.com>

### CODE REVIEW

File: tools/csv-merger/csv-merger.go [hunk 1]
```diff
@@ -931,6 +933,7 @@ func getMetricsExporterDeployment() appsv1.DeploymentSpec {
 					},
 				},
 				ServiceAccountName: "ocs-metrics-exporter",
+				Tolerations:        ocsNodeToleration,
```
Comment (umangachapagain) [inline tools/csv-merger/csv-merger.go:936]: We don't need this.

File: tools/csv-merger/csv-merger.go [hunk 2]
```diff
@@ -502,14 +511,7 @@ func generateUnifiedCSV() *csvv1.ClusterServiceVersion {
...
 		d := &templateStrategySpec.DeploymentSpecs[i]
-		d.Spec.Template.Spec.Tolerations = []corev1.Toleration{
-			{
-				Key:      "node.ocs.openshift.io/storage",
-				Operator: corev1.TolerationOpEqual,
-				Value:    "true",
-				Effect:   corev1.TaintEffectNoSchedule,
-			},
-		}
+		d.Spec.Template.Spec.Tolerations = ocsNodeToleration
```
Comment (umangachapagain) [inline tools/csv-merger/csv-merger.go:514]: We should move the entire loop below i.e. after we append metrics-exporter deployment to templateStrategySpec. That should give the same effect without code duplication.

**Review by umangachapagain — CHANGES_REQUESTED** [review]

File: tools/csv-merger/csv-merger.go (hunk 2 above)
Comment (jarrpa) [reply tools/csv-merger/csv-merger.go]: ...oh, duh! Changed.

**Review by umangachapagain — APPROVED** [review]

**Comment by jarrpa** [comment]
@umangachapagain Seems the change I submitted to openshift/release went through, we now need explicit /lgtm again. :)

**Pull request merged**